### PR TITLE
Incorporation of new RTS files for new Long Term ECS SI modes

### PR DESCRIPTION
**Pull request by Gregg140**
## Description

New variations of SI modes, and their associated RTS files,  for Long Term ECS
measurements were created to provide a more flexible chip selection.
Backstop History was updated to incorporate these new RTS files in future
history assemblies.

The added RTS files are:

1_ECS2.RTS
1_ECS3A.RTS
1_ECS3B.RTS
1_ECS5.RTS
1_ECS6.RTS

Changes to the python files consist mainly of documentation changes. The major
portion of the update is to include the new RTS files in the directory and an update to a
data file.

## Interface impacts

None

### Unit tests

- [ ] No unit tests
- [ ] Mac
- [ x] Linux
- [ ] Windows

Unit and regression testing completed, using all new and operational RTS files,
to assure that the assembled histories correctly contained the commands
from the new RTS files whenever they were used.

Unit tests included:

- Isolated tests of processRTS which reads and process an RTS file to assure the new files
  were correctly read and processed by the code.

- An assembly of the history of the SEP0917 load after an interrupt of the SEP0417
load to assure the assembled history contained the correct RTS commands

### Functional tests

Regression testing Loads were full LR runs with checks of the thermal model outputs
and assembled history files to assure correct processing of the new RTS files. Thermal
model outputs were compared to the production load output to check the resultant
change in the thermal response where appropriate.

Loads used for Regression testing:

 - FEB2023B Maneuver-Only load executed after the Full Stop interruption of
   the FEB1323 load.

 - FEB2423A Return to Science load which is a TOO interruption of FEB2023.
 
 - All tests passed.

Objectives and changes reviewed by ACIS Ops.

### CODE REVIEW

**Review by jazan12 — APPROVED** [review]
All looks good to me!

File: backstop_history/CR_Commands.dat [hunk 1]
```diff
@@ -34,10 +34,21 @@
  "WSPOW00F1E": "1999:001:00:00:00.000 |       20 0 | ACISPKT          | TLMSID= WSPOW00F1E, CMDS= 5, WORDS= 7, PACKET(40)= D0000000000000000000000000000000000     , SCS= 107, STEP= 1",
  "WSPOW1E01E": "1999:001:00:00:00.000 |       20 0 | ACISPKT          | TLMSID= WSPOW1E01E, CMDS= 5, WORDS= 7, PACKET(40)= D0000000000000000000000000000000000     , SCS= 107, STEP= 1",
 
- "OORMPDS": "1999:001:00:00:00.000 |       20 0 | COMMAND_SW       | TLMSID= OORMPDS, HEX= 800030C, MSID= OORMPDS, SCS= 107, STEP= 1",
+  "OORMPDS": "1999:001:00:00:00.000 |       20 0 | COMMAND_SW       | TLMSID= OORMPDS, HEX= 800030C, MSID= OORMPDS, SCS= 107, STEP= 1",
```
Comment (jzuhone) [inline backstop_history/CR_Commands.dat]: Don't know if the space alignment matters here or not

File: backstop_history/LTCTI_RTS.py [hunk 2]
```diff
@@ -177,16 +183,16 @@ def processRTS( RTS_name, SCS_NUM, NUM_HOURS, RTS_start_date):
...
 
     RTS_dtype = [('date', '|U20'),
-                 ('time','<f8'),
-                 ('statement', '|U20'),           
-                 ('mnemonic', '|U20'), 
-                 ('substitution_parameter',  '|U20'),
-                 ('substitution_parameter_value',  '|U20'),
-                 ('DELTA','<f8'),
-                 ('SCS_NUM', '|U5')]
+                          ('time','<f8'),
```
Comment (jzuhone) [inline backstop_history/LTCTI_RTS.py:186]: no big deal, but something went wrong with the whitespace here

**Review by jzuhone — APPROVED** [review]
I noticed some issues with whitespace, but otherwise no problems.

File: backstop_history/CR_Commands.dat (hunk 1 above)
Comment (Gregg140) [reply backstop_history/CR_Commands.dat]: Didnt matter but it's fixed.

File: backstop_history/LTCTI_RTS.py (hunk 2 above)
Comment (Gregg140) [reply backstop_history/LTCTI_RTS.py:186]: Nothing wrong with the whitespace. 7 of the items in the DTYPE specification were moved over to align under "date"

**Comment by taldcroft** [comment]
@Gregg140 - I have some questions about testing before I release the FSDS ticket for review.

Can you elaborate on the unit testing?  
- How did you perform the isolated tests of processRTS for each RTS and compare to the expected output?
- How did you generate a new assembled history for each of the new RTS's.
- How did you determine that the new assembled history matched the expected commands from the RTS.
- Where is this documented (in a perfect world).

**Comment by Gregg140** [comment]
It is documented in an update to the Release Notes and the answers to
your questions
are contained in the update

**Comment by taldcroft** [comment]
@Gregg140 - thanks for the update, looks good now.

**Pull request merged**